info.yml

```diff
@@ -122,6 +122,24 @@ configuration:
         values:
             type: dict
 
+    published_file_matching_fields:
```

Comment:
Any better name? Something including "hierarchy" may be? Or "grouping"?

Reply:
After seeing the code, it could be something with "history" in it. `publish_history_by_fields`?